### ISSUE TITLE
Missing "Skip to Main Content" / "Skip to nav" links

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#2426]
## Summary

Both Admins and Talent Search applications are missing a link to allow assistive technology to skip the navigation and be directed to the main content. This is more of an issue for the Admin application as the main navigation is fairly significant.

Ref: https://webaim.org/techniques/skipnav/

## Proposed Solution

This would be as simple as adding a hidden link to a div wrapping the main content.

### Example

```jsx
<a class="sr-only" href="#main">Skip to main content</a>
<Header />
<main id="main">
  {content}
</main>
<Footer />
```
